extra-multimedia/alsa-utils/autobuild/prepare

```diff
@@ -0,0 +1 @@
+autoreconf -fi
```

Comment:
Missing abinfo, verbosity. Also, why would this be necessary, was RECONF disabled?